.github/workflows/push-pull-main.yml

```diff
@@ -38,36 +38,7 @@ on:
         required: false
 
 jobs:
-  commit-updated-env:  # Keep envs read by external sources (binder and readthedocs) up-to-date
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          token: ${{ secrets.DEPENDABOT_WORKFLOW_TOKEN }}
-          ref: ${{ github.event.pull_request.head.ref }} # Check out the head of the actual branch, not the PR
-          fetch-depth: 0 # otherwise, you will fail to push refs to dest repo
-      - uses: pyiron/actions/write-docs-env@main
-      - uses: pyiron/actions/write-environment@main
-        with:
-          env-files: ${{ inputs.notebooks-env-files }}
-          output-env-file: .binder/environment.yml
-      - name: commit
-        id: commit-docs-env
-        continue-on-error: true  # Allow "failure" when there are no changes (the working tree is clean)
-        run: |
-          git config --local user.email "pyiron@mpie.de"
-          git config --local user.name "pyiron-runner"
-          git add docs/environment.yml .binder/environment.yml
-          git commit -m "Update env file" -a
-      - name: push
-        if: steps.commit-docs-env.outcome == 'success'  # But only push if we made it here without continue-on-error
-        uses: ad-m/github-push-action@master
-        with:
-          github_token: ${{ secrets.DEPENDABOT_WORKFLOW_TOKEN }}
-          branch: ${{ github.event.pull_request.head.ref }}
-
   build-docs:
-    needs: commit-updated-env
```

Comment:
If we keep the job, and we should, then this is useful because it stops us from burning CI time if there is a new commit incoming. The day to starting the rest of the CI is minimal so I'm not worried about wasting human time either.